bac_index.qmd

```diff
@@ -15,8 +15,8 @@ toc-depth: 4
 ### Graphs
 
 -   [AG -- Utilities for All Graphs](./graphs/ag01.qmd)
--   [DG -- Histograms, Barplots, Scatterplots](./graphs/dg01.qmd)
--   [KG -- Kaplan-Meier Graphs](./graphs/kg01.qmd)
+-   [DG -- Histograms, Barplots, Scatterplots](./graphs/DG1/dg01.qmd)
```

Comment:
I know we merged this like this before already, but I wonder if it would be better to go back to more detailed index. e.g. in the current published version on https://insightsengineering.github.io/biomarker-catalog/bac_index.html we see all the main variants.

I think this high level grouping is still useful. E.g. I think below `DG` we could have a lower level bullets that describe the 4 main variants of DG (and without going into the potential variants A, B etc. of these templates)

As we are thinking about this hierarchy, it might even make sense for the navigation page to be hierarchical too. So put `DG` as first level, then `DG1` to `DG4` , and potentially have variants underneath. So that would make it 3 levels. The benefit would be that the user can see on a single screen all level 1 index items and can then drill down.

Reply:
ok I will try to implement this by the next commit

Reply:
@danielinteractive please review the proposal and let me know if it fulfills the request and resolves the conversation

Reply:
@danielinteractive please review the proposal for the bac_index.qmd, as commented in the commit, the high level links to the first sub template in that category, because there is no specific page (eg. CMT links to CMT1)